web/main.ts

```diff
@@ -4,7 +4,9 @@
 /// <reference lib="dom.asynciterable" />
 /// <reference lib="deno.ns" />
 
-import "std/dotenv/load.ts";
+import { load } from "std/dotenv/mod.ts";
+
+const env = await load();
```

Comment:
I think it's better to use combinedEnv in /runHelpers.ts or `const env = {...(await load()), ...Deno.env.toObject()}` because it should be easier to use if we can override .env with `WEBUI_SESSION_APP_KEY=something deno run -A web/main.ts`. (Also with docker images where you can easily change the environment variable but not .env.)